modules/ROOT/pages/dw-dataformat-types.adoc

```diff
@@ -1,57 +1,51 @@
-== DataFormat Types (dw::extension::DataFormat)
+= DataFormat Types (dw::extension::DataFormat)
 
-The DataFormat module contains what you need to register a new data format for
-the DataWeave language. For an example, see
-https://github.com/mulesoft-labs/data-weave-custom-data-format[Custom Data Formats Example].
-
-[%headers, cols="1,3,6"]
 |===
 | Type | Definition | Description
 
 | DataFormat
-| `type DataFormat = { binaryFormat?: Boolean, defaultCharset?: String,
-fileExtensions?: Array<String&#62;,
-acceptedMimeTypes: Array<MimeType&#62;,
-reader: &#40;content: Binary,
-charset: String,
-settings: ReaderSettings&#41; &#45;&#62; Any,
-writer: &#40;value: Any,
-settings: WriterSettings&#41; &#45;&#62; Binary }`
-a| Represents the DataFormat definition.
-
-* `binaryFormat?: Boolean`: True if this is data format is represented in a
-binary representation instead of text. If not present, false.
-* `defaultCharset?: String`: The default charset of this format, if any.
-* `fileExtensions?: Array<String&#62;`: Returns the list of file extensions
-with the . &#40;".json", ".xml", etc...&#41; that should be assigned to this
-Data Format
-* `acceptedMimeTypes: Array<MimeType&#62;` The list of MimeTypes that are
-accepted.
-* `reader: (content: Binary, charset: String, settings: ReaderSettings) -> Any`:
-This function will be in charge of reading the raw content and transform it
-into the DataWeave canonical model.
-* `writer: (value:Any, settings:WriterSettings) -> Binary`: This function will
-be in charge of writing the DataWeave canonical model into Binary content.
+| `type DataFormat = { /**
+* True if this is data format is represented in a binary representation instead of text, if not present is false
+**/
+binaryFormat?: Boolean, /**
+* The default charset of this format, if any.
+**/
+defaultCharset?: String, /**
+* Returns the list of file extensions with the . &#40;".json", ".xml", etc...&#41; that should be assigned to this Data Format
+**/
+fileExtensions?: Array<String&#62;, /**
+* The list of MimeTypes that are accepted
+**/
+acceptedMimeTypes: Array<MimeType&#62;, /**
+* This function will be in charge of reading the raw content and transform it into the DW canonical model
+**/
+reader: &#40;content: Binary, charset: String, settings: ReaderSettings&#41; &#45;&#62; Any, /**
+* This function will be in charge of writing the DW canonical model into Binary content
+**/
+writer: &#40;value: Any, settings: WriterSettings&#41; &#45;&#62; Binary }`
+| 
 
 
 | EmptySettings
 | `type EmptySettings = Object`
-| Represents a configuration with no Settings.
```

Comment:
MDA: Not sure if the EmptySettings and most other type output was autogenerated. The comments are placed in a different location than compared to EncodingSettings. And EncodingSettings here is different from source.